87.md

```diff
@@ -0,0 +1,151 @@
+NIP-87
+=======
+
+Closed Groups
+------------------
+
+Some online communities may want a measure of privacy. While nostr's architecture isn't conducive to high levels of privacy due to metadata leakage and the difficulty of implementing ratchets for key rotation, it can provide basic encrypted channels appropriate for use with large groups with dynamic member lists.
+
+This NIP relies on [./86.md](NIP 86) for key sharing.
```

Comment:
can someone point me to nip-86? couldn't find it

Reply:
https://github.com/nostr-protocol/nips/pull/876

Reply:
This was a leftover, key sharing is now included in 87.